go.mod

```diff
@@ -44,6 +46,7 @@ require (
 	github.com/otiai10/copy v1.0.2
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
 	github.com/pkg/errors v0.9.1
+	github.com/replicatedhq/kurl/kurlkinds v0.0.0-20200227234931-c098073f445c
```

Comment:
this should probably be done off of tags I think?

Reply:
i agree, we do tag every release https://github.com/replicatedhq/kURL/releases